src/topology/instances/ereal.lean

```diff
@@ -0,0 +1,355 @@
+/-
+Copyright (c) 2021 Sébastien Gouëzel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sébastien Gouëzel
+-/
+import topology.instances.ennreal
+import data.real.ereal
+
+/-!
+# Topological structure on `ereal`
+
+We endow `ereal` with the order topology, and prove basic properties of this topology.
+
+## Main results
+
+* `coe : ℝ → ennreal` is an open embedding
+* `coe : ℝ≥0∞ → ennreal` is an embedding
```

Comment:
are these two coe supposed to be to ereal instead of ennreal?

Reply:
Good catch, thanks! Fixed.